src/application/controllers/property/router.ts

```diff
@@ -21,6 +21,11 @@ export function PropertyRouter() {
       StatusCodes.CREATED,
     ),
   );
+  router.get(
+    '/myProperties',
```

Comment:
I would rename it to something like 'owned' so that the final endpoint looks like '/property/owned', it makes more sense.